<launch>

    <!-- Bringup robot from the ssh terminal -->
    <remap from="/base_scan" to="/scan" />

    <!-- Driving the robot -->
	<node pkg="hw1" name="simple_move" type="simple_move.py" output="screen">

    <!-- Maximum robot speed in the body x direction -->
    <param name="v_max" type="double" value="-0.1"/>
    <!-- Anglur range of laser scan reading considered for obstacle detection -->
    <param name="max_angle" type="int" value="5"/>
    <!-- Maximum stopping distance (along body x) -->
    <param name="range_lim" type="double" value="0.16"/>
    <!-- Sim update rate -->
    <param name="sample_rate" type="int" value="10"/>
    </node>

</launch>